worlds/stardew_valley/strings/ap_names/mods/mod_items.py

```diff
@@ -10,6 +10,7 @@ class DeepWoodsItem:
 
 class SkillItem:
```

Comment:
Call the class "SkillLevel" and then the members can be "luck" and "Archaeology", without repeating "level" everywhere. Plus, now it's inconsistent (luck_skill and archaeology_level?). This will make it consistent